models/issue_mail.go

```diff
@@ -42,6 +42,11 @@ func mailIssueCommentToParticipants(issue *Issue, doer *User, comment *Comment,
 		participants = append(participants, issue.Poster)
 	}
 
+	// Assignee must receive any communications
+	if issue.AssigneeID != doer.ID {
+		participants = append(participants, issue.Assignee)
```

Comment:
It have not checked too deeply but if there is error in issue `loadAttributes`, it would probably continue and leave issue.Assignee as nil. I would suggest adding additional check for ` && issue.Assignee != nil` to be on safe side just in case